File: TP/TP1/Code/5G.cpp
/* Afficher une ligne de longueur N, alternant les étoiles et les espaces */
#include <iostream>

int main()
{
  int n;
  
  std::cout << "Quelle est la longueur de la ligne ?" << std::endl;
  std::cin >> n;

  for (int i = 0; i < n; ++i) {
    if (i % 2 == 0)
      std::cout << "*" << std::flush;
    else
      std::cout << " " << std::flush;
  }

  std::cout << std::endl;
  return 0;
}

File: TP/TP1/Code/Python/5G.py
#!/usr/bin/python
# Afficher une ligne de longueur N, alternant les étoiles et les espaces
print('Donne moi une valeur : ', end='')
foo = int(input())

for i in range(foo):
    if i % 2 == 0:
        print('*', end='')
    else:
        print(' ', end='')
